src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -162,9 +162,11 @@ const exprt java_bytecode_convert_methodt::variable(
   size_t address,
   java_bytecode_convert_methodt::variable_cast_argumentt do_cast)
 {
-  irep_idt number=to_constant_expr(arg).get_value();
+  mp_integer number;
+  if(to_integer(to_constant_expr(arg), number))
+    throw "failed to extract number";
```

Comment:
Is this a user-facing error or an invariant/precondition? (same question regarding other throws below)